lib/steps.js

```diff
@@ -396,109 +396,17 @@ When(/I tap the (button|link) "([^"]+)"/, async (elemType, phrase) => {  // √
 // -------------------------------------
 // --- VARIABLES-BASED STEPS ---
 
-When(/I set the var(?:iable)? "([^"]+)" to "([^"]+)"/, async (var_name, value) => {
-  /* Use variable names and values to interact with:
-  *  - Buttons that have values
-  *  - Input elements that take text as input
-  *  - Textareas
```

Comment:
I think we should still mention that this function covers interactions with buttons, text input, textareas, and radios.

Reply:
Good idea. I think I may have moved that to scope.js along with the related code.